swagger_to/style.py

```diff
@@ -227,31 +232,39 @@ def check_description(description: str) -> Optional[str]:
     trimmed = description.replace("|", "")
 
     if not trimmed[:1].isalpha():
-        return "should start with alphanumeric letter: {}".format(description)
+        return "should start with alphanumeric letter"
 
     if trimmed[:1].isupper():
-        return "should start with lower case letter: {}".format(description)
+        return "should start with lower case letter"
 
     words = trimmed.split(' ')
 
     if not words[0].endswith('s'):
-        return "should start with verb in present tense: {}".format(description)
+        return "should start with verb in present tense"
 
     lines = trimmed.splitlines()
 
     if not lines[0].endswith('.'):
-        return "first line should end with a period: {}".format(description)
+        return "first line should end with a period"
+
+    one_empty_line = False
     for i, word in enumerate(lines):
         if i == 1 and word != "":
-            return "second line should be empty: {}".format(description)
-        elif i != 1 and word == "":
-            return "no line apart from the second should be empty: {}".format(description)
+            return "second line should be empty"
+
+        if word == "":
+            if one_empty_line:
+                return "two empty lines are not allowed"
+            else:
+                one_empty_line = True
+        else:
+            one_empty_line = False
 
     if not trimmed.endswith("."):
-        return "should end with a period: {}".format(description)
+        return "should end with a period"
 
     if trimmed.strip() != trimmed:
```

Comment:
I'd move this to the top and check that no line contains a trailing white space.